src/components/SlateEditor/plugins/embed/EditVideoTime.tsx

```diff
@@ -28,8 +27,6 @@ const hmsCSS = css`
 
 interface Props {
   name: string;
-  src: string;
-  onFigureInputChange: (e: Event) => void;
   t: TranslateType;
```

Comment:
den `t`'en skal ikke lenger defineres i props, den skal hentes fra `import { injectT, tType } from '@ndla/i18n';` og bli brukt enten sånn
```
const EditVideoTime: FC<Props & tType> = (props: Props) => {
```

eller sånn

```
const EditVideoTime = ({props}: Props & tType) => {
```

Hvis t'en er definert i Props på den måten som nå, så vil komponenten kreve at t'en skal bli sendt som en prop, men vi injecter t'en via injectT() funksjonen, så vi ønsker at komponenten skal være uanvengig av t prop'en. 

Tror du kanskje må pulle fra master, for jeg ser nå at jeg fiksa alle stedene som har `TranslateType`